marc.go

```diff
@@ -73,33 +73,58 @@ func (m *MarcGenerator) Generate() <-chan Record {
 
 func (m *marcparser) parse(out chan Record) {
 	mr := fml.NewMarcIterator(m.file)
+	var errorCount int
 
 	for mr.Next() {
 		record, err := mr.Value()
 
 		if err != nil {
 			log.Println("Error parsing MARC record:", record.ControlNum())
 			log.Println(err)
-			log.Println(record)
+			// os.Stderr.WriteString("--- Begin Problem MARC Record ---\n")
```

Comment:
I left this in but commented out because it was really useful but shouldn't be on by default and I couldn't think of an easy way to make a switch for it. I can remove it if bothers anyone to have commented out code.